frontend/src/styling/AddKpi.css

```diff
@@ -18,12 +18,12 @@ form {
 .selector {
     width: 100px;
     height: 20px;
-    margin: 20px;
+    margin: 10px;
 }
 
 input {
```

Comment:
versuche classnames zu verwenden , falls du andere Input felder in den fall verwendest werden die natürlich auch angepasst . Meine Persönliche methode ist die häufigere Verwendung von Classname